pkg/configuration/configuration.go

```diff
@@ -110,92 +110,92 @@ type ClusterNamespaces map[string]string
 
 // LoadClusterAccessDefinition loads ClusterAccessDefinition object from the config file and checks that all required parameters are set
 func LoadClusterAccessDefinition(term ioutils.Terminal, clusterName string) (ClusterAccessDefinition, error) {
-	sandboxUserConfig, _, err := Load(term)
+	ksctlConfig, _, err := Load(term)
```

Comment:
`cfg` would be enough here. It's a shorter name, and I guess that the `ksctl` prefix is superfluous here

Reply:
that variable name is used at multiple places, I would keep it as it is to make it consistent and self-explanatory. 
if you insist on making it shorter, I could rename the occurrences to `ksctlCfg`, but I'm not sure if 3 chars extra is a big deal.

Reply:
as I said, I don't really see the value of the `ksctl` prefix but I will not insist on this ;)